_includes/specials/taproot/en/15-output-linking.md

```diff
@@ -0,0 +1,57 @@
+{% auto_anchor %}
+
+After taproot activates, users will begin receiving payments to P2TR
+outputs.  Later, they'll spend those outputs.  In some cases, they'll
+make payments to non-P2TR outputs but will still return their change to
+themselves using a P2TR change output.
+
+{:.center}
+![Example transaction P2TR -> {P2WPKH, P2TR}](/img/posts/2021-10-p2tr-to-p2tr_p2wpkh.png)
+
+It's easy for an expert or an algorithm observing such a transaction to
+reasonably infer that the P2TR output is the user's own change output,
+making the other output the payment output.  This isn't guaranteed to be
+true, but it is the most likely explanation.
+
+Some have argued that the many [privacy benefits][p4tr benefits] of taproot should be
+ignored because of this possible temporary decrease in privacy during the
+transition of wallets to P2TR.  Many experts have [called][coindesk
+experts] that an unwarranted overreaction.  We agree and can also offer
+a few additional counterpoints to consider:
+
+- **Other metadata:** transactions may contain other metadata which
+  reveals which outputs are change and which are payments.  One of the
```

Comment:
Consider:

```suggestion
- **Other metadata:** transactions may contain other metadata that
  reveals which outputs are change and which are payments.  One of the
```

to avoid the repeated "which"